app/models/ability.rb

```diff
@@ -7,8 +7,15 @@ def initialize(user)
     # Define abilities for the passed in user here. For example:
     #
     @user = user || User.new # guest user (not logged in)
+
+    can :manage, Availability
+    cannot :place, Availability
+    cannot :show_everyone, Availability
+    cannot :manage, Timeslot
```

Comment:
You do not need the `cannot :manage, Timeslot` and `cannot :show_user_console, User` here. By default they cannot